python/pyspark/sql/tests/test_functions.py

```diff
@@ -1029,6 +1034,11 @@ def test_np_scalar_input(self):
             res = df.select(array_position(df.data, dtype(1)).alias("c")).collect()
             self.assertEqual([Row(c=1), Row(c=0)], res)
 
+    def test_binary_math_function(self):
+        for func, res in [(atan2, 0.13664), (hypot, 8.07527), (pow, 2.14359), (pmod, 1.1)]:
+            df = self.spark.range(1).select(round(func(1.1, 8), 5).alias("a"))
+            self.assertEqual(Row(a=res), df.first())
```

Comment:
Nit. We could skip multiple actions here, doing something around these lines (not tested, there might be some typos there):


```python
funcs, expected = zip(*[(atan2, 0.13664), (hypot, 8.07527), (pow, 2.14359), (pmod, 1.1)])
df = spark.range(1).select(*(func(1.1, 8) for func in funcs)

for a, e in zip(df.first(), expected):
    self.assertAlmostEqual(a, e, 5)
```

Reply:
@khalidmammadov if you want to have a try, feel free to send a followup PR

Reply:
> oops, I did see this comment when I was merging this ...

No worries @zhengruifeng  :)

Reply:
Sure, will send followup PR